index.bs

```diff
@@ -2294,10 +2314,21 @@ The following {{DOMException}} exceptions can be raised:
     :   {{UnknownError}}
     ::  The [=authenticator=] could not process the supplied options,
         or encountered an error while creating the new credential.
+    
+    :   {{UserCancellationError}}
+    ::  The user has exercised a user agent user-interface option
+        to end the ceremony.
+
+    :   {{UserHybridCancellationError}}
+    ::  The user has exercised a user agent user-interface option
```

Comment:
In most cases, hybrid is not handled by the user agent. Is this error only for the cases where the user agent implements hybrid directly?

Reply:
I suppose yes, Chrome's prompt is what I had in mind when I was thinking about this:

![Screenshot 2024-08-07 at 11 43 36 AM](https://github.com/user-attachments/assets/92fb672c-c85a-44dd-8206-4f9912a7ff9f)

Could a blocker to codifying an error like `UserHybridCancellationError` be that when the platform handles hybrid it might not tell the user agent that the user cancelled out of prompt to try hybrid? 🤔

Reply:
I'm taking `UserHybridCancellationError` out of the running for now, I don't want it to block some of the other errors.